### DISCUSSION TITLE
is multicore cpu render possible?

### DISCUSSION THREAD

**Discussion by handlingcfg** [JHRobotics/softgpu#81] [Ideas]
i'm curious about whether or not using multicore like simd95 with multicore assembly,

right now llvmlipe gets about 40 fps in 3dm99 game1, 3d accelerated version gets about 100 fps,if its possible, in theory it can deliver more than vmware svga2

**Comment by Torinde** [comment]
Multicore DOS: https://github.com/WindowsNT/asm
Multicore Win9x: RLoew [MULTCORE](https://msfn.org/board/topic/180215-rloew-1952-2019-has-passed-away/page/7/)
